tests/system/src/test/java/test/javafx/scene/InitializeJavaFXTest.java

```diff
@@ -0,0 +1,84 @@
+package test.javafx.scene;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.stage.Stage;
+import junit.framework.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+public class InitializeJavaFXTest {
+
+    public static final CountDownLatch appLatch = new CountDownLatch(1);
+
+    public static class InitializeApp extends Application {
+        @Override
+        public void start(Stage primaryStage) throws Exception {
+            appLatch.countDown();
+        }
+    }
+
+    public static void initializeApplication() throws Exception {
```

Comment:
This method is unused, along with the `InitializeApp` class. Did you plan to use it?

Reply:
It's useful to compare it with the behavior of the two methods to start JavaFX.
If it would be my codebase, I would keep it, so if someone investigates it later, it's easier to investigate for differences. But I can also delete it, what would you prefer?

Reply:
It's now used because i added more test cases.